### PR TITLE
Add claims request support on token endpoint requests

### PR DESCRIPTION
**Pull request by pkanher617**
This PR adds the option to pass a stringified claims object to the `/token` endpoint.

### CODE REVIEW

File: lib/msal-common/test/client/AuthorizationCodeClient.spec.ts [hunk 1]
```diff
@@ -335,7 +335,8 @@ describe("AuthorizationCodeClient unit tests", () => {
                 scopes: [...TEST_CONFIG.DEFAULT_GRAPH_SCOPE, ...TEST_CONFIG.DEFAULT_SCOPES],
                 redirectUri: TEST_URIS.TEST_REDIRECT_URI_LOCALHOST,
                 code: TEST_TOKENS.AUTHORIZATION_CODE,
-                codeVerifier: TEST_CONFIG.TEST_VERIFIER
+                codeVerifier: TEST_CONFIG.TEST_VERIFIER,
+                claims: "{claimsRequest}"
```
Comment (jasonnutter) [inline lib/msal-common/test/client/AuthorizationCodeClient.spec.ts:339]: Is this what a typical claims request looks like? I'd be interested for a real-world example (out of curiosity).

**Comment by jasonnutter** [comment]
Can you add a milestone please?

File: lib/msal-common/test/client/AuthorizationCodeClient.spec.ts (hunk 1 above)
Comment (AndreiZe) [reply lib/msal-common/test/client/AuthorizationCodeClient.spec.ts:339]: Usually it is serialized JSON object. Here is how claims request looks like when supporting Long Lived Tokens:
"{"access_token":{"xms_cc":{"values":["CP1"]}}}"

File: lib/msal-common/src/request/RefreshTokenRequest.ts [hunk 2]
```diff
@@ -14,4 +14,5 @@ import { BaseAuthRequest } from "./BaseAuthRequest";
  */
 export type RefreshTokenRequest = BaseAuthRequest & {
     refreshToken: string;
+    claims?: string;
```
Comment (tnorling) [inline lib/msal-common/src/request/RefreshTokenRequest.ts:17]: Should we make this type `object` in the Request objects and stringify when adding it to the request?

File: change/@azure-msal-common-2020-08-20-15-48-02-claims-request-tokenendpt-v2.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Add claims request to /token calls (#2138)",
+  "packageName": "@azure/msal-common",
+  "email": "thomas.l.norling@gmail.com",
```
Comment (jmckennon) [inline change/@azure-msal-common-2020-08-20-15-48-02-claims-request-tokenendpt-v2.json:5]: We decided these should be our Microsoft emails, right?

Comment (tnorling) [reply change/@azure-msal-common-2020-08-20-15-48-02-claims-request-tokenendpt-v2.json:5]: Thanks. I dont know why it's still doing this. I changed it and I guess it changed itself back

**Pull request merged**